ledger/src/lib.rs

```diff
@@ -13,6 +13,7 @@ pub mod blockstore;
 pub mod ancestor_iterator;
 pub mod blockstore_db;
 pub mod blockstore_meta;
+pub mod blockstore_metric;
```

Comment:
nit: "blockstore_metrics" (plural) instead of "blockstore_metric". That is, can you rename the new file to `blockstore_metrics.rs` instead of `blockstore_metric.rs`